src/trace/log.zig

```diff
@@ -141,8 +142,9 @@ pub const ChannelPrintLogger = struct {
             .exit = AtomicBool.init(false),
             .max_level = config.max_level,
             .handle = null,
-            .channel = try Channel([]const u8).create(config.allocator),
+            .channel = try Channel([]const u8).init(config.allocator),
```

Comment:
why is this not `create`?

Reply:
The logger itself is already a self-managed allocation. It's not necessary to do nested allocations for its fields.